### DISCUSSION TITLE
How do I report a bug in the docs?

### DISCUSSION THREAD

**Discussion by electrofloat** [authelia/authelia#4810] [Q&A] [answered]
I tried to create an issue, but the form requires a lot of information that is irrelevant to the doc bug.

So the docs here: https://github.com/authelia/authelia/blob/master/docs/content/en/configuration/prologue/common.md
 describes the server idle timeout as: "Configures the server write timeout."

it is probably just a copy-paste bug, and I could also just edit it myself, but since I don't know the intention (and the code) it would be just way too lazy to replace write with idle.

(and just to praise the guys working on this, although I've only finished the overview section and just starting with the configuration, it was very long time ago when I last saw such a good documentation for a project. It is very-very informative, describing everything thoroughly. Bravo!)

**Comment by james-d-elliott** [comment, marked as answer]
Yeah it's a typo should be idle as you thought, and you can either report it through discussions or just edit it yourself.
